file-reading.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3.9
```

Comment:
A la hora de importar el fichero en `main.py` la importación da problemas por el guión que hay en el nombre:

https://pyphilly.org/hyphen-and-underscores-when-use-each-django-and-python/#:~:text=Python%20packages%20and%20modules%20can,name%20if%20it%20improves%20readability.

Reply:
he cambiado el guión por una guión bajo para que no de problemas